src/core/gui/sidebar/previews/base/SidebarPreviewBaseEntry.cpp

```diff
@@ -118,11 +125,26 @@ void SidebarPreviewBaseEntry::paint(cairo_t* cr) {
         doRepaint = true;
     }
 
+    GtkAllocation alloc;
+    gtk_widget_get_allocation(widget, &alloc);
+    int bufferWidth = cairo_image_surface_get_width(crBuffer);
+    int bufferHeight = cairo_image_surface_get_height(crBuffer);
```

Comment:
Is using the allocation here really useful? Wouldn't comparing the size to
```
    double height = page->getHeight() * sidebar->getZoom();
    double width = page->getWidth() * sidebar->getZoom();
```
be cleaner?

Reply:
I think it matters, but I will check once more.